README.md

```diff
@@ -8,4 +8,18 @@ The Ziggurat implementation for Algorand's `algod` nodes.
 
 1. Clone this repository.
 2. Build [go-algorand](https://github.com/algorand/go-algorand) from source.
-3. TBD
+3. Run setup script:
+```zsh
+    ./tools/setup_env.sh
+```
+
+In case algorand files are installed in a specific location, please export that location to the `ALGORAND_BIN_PATH`
```

Comment:
```suggestion
In case algorand files are installed in a specific location, export that location to the `ALGORAND_BIN_PATH`
```

Reply:
I should stop being nice to people 😀
Done